### ISSUE TITLE
Error when using shapelet decomposition

### ISSUE THREAD

**Issue by shulevski** [lofar-astron/PyBDSF#94]
Using 

> shapelet_do=True

in `process_image`, I get an error:

```
Decomposing islands into shapelets ...... : [Thread raised exception math domain error-----------------------------------------------------------------------------------------------] 2/730
Traceback of thread is:
-------------------------
  File "/usr/local/lib/python2.7/dist-packages/bdsf-1.9.0-py2.7-linux-x86_64.egg/bdsf/multi_proc.py", line 54, in worker
    result = f(val)
  File "/usr/local/lib/python2.7/dist-packages/bdsf-1.9.0-py2.7-linux-x86_64.egg/bdsf/functions.py", line 2058, in eval_func_tuple
    return f_args[0](*f_args[1:])
  File "/usr/local/lib/python2.7/dist-packages/bdsf-1.9.0-py2.7-linux-x86_64.egg/bdsf/shapefit.py", line 98, in process_island
    (beta, centre, nmax) = self.get_shapelet_params(arr, mask, basis, beam_pix, fixed, N.array(isl.origin), mode)
  File "/usr/local/lib/python2.7/dist-packages/bdsf-1.9.0-py2.7-linux-x86_64.egg/bdsf/shapefit.py", line 124, in get_shapelet_params
    beta=sqrt(m3[0]*m3[1])*2.0
-------------------------
 Bad input params
Thread raised exception list indices must be integers, not tuple
Traceback of thread is:
-------------------------
  File "/usr/local/lib/python2.7/dist-packages/bdsf-1.9.0-py2.7-linux-x86_64.egg/bdsf/multi_proc.py", line 54, in worker
    result = f(val)
  File "/usr/local/lib/python2.7/dist-packages/bdsf-1.9.0-py2.7-linux-x86_64.egg/bdsf/functions.py", line 2058, in eval_func_tuple
    return f_args[0](*f_args[1:])
  File "/usr/local/lib/python2.7/dist-packages/bdsf-1.9.0-py2.7-linux-x86_64.egg/bdsf/shapefit.py", line 100, in process_island
    cf = decompose_shapelets(arr, mask, basis, beta, centre, nmax, mode)
  File "/usr/local/lib/python2.7/dist-packages/bdsf-1.9.0-py2.7-linux-x86_64.egg/bdsf/shapelets.py", line 40, in decompose_shapelets
    B = shapelet_image(basis, beta, centre, hc, coord[0], coord[1], image.shape)
  File "/usr/local/lib/python2.7/dist-packages/bdsf-1.9.0-py2.7-linux-x86_64.egg/bdsf/shapelets.py", line 104, in shapelet_image
```
The image I work on is an AARTFAAC-12 all-sky image, but it does not have any unusual properties.

**Comment by darafferty** [comment]
I'm not entirely sure what caused the problem, but I added a default value for beta that might fix it. Can you update to the latest master branch and try again?

**Comment by darafferty** [comment]
This issue seems to be solved with commit 0f9d80a.
